docs/guides/platform/Install_MERLIN_Online.md

```diff
@@ -172,13 +171,13 @@ EOF
 
 After a few minutes, the operator is installed. Verify that the all components are in the Succeeded state by running the following command:
```

Comment:
If 'Succeeded' is the correct response that users should look for, please put it in backticks.
Why is the command now 'ibm-merlin'?  That would not match 'ibmi-merlin' operator.

Reply:
Done. The change is made to use backticks. 
About the command, it was wrong even in doc for v1. `ibmi-merlin-operator` is the package name for the operator bundles, csv name if of format `ibm-merlin.v<version>`. So we need to use the latter one to get status of csv for merlin operator.